core/src/main/java/org/stellar/anchor/client/TempClient.java

```diff
@@ -0,0 +1,54 @@
+package org.stellar.anchor.client;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.Set;
+import lombok.*;
+
+/**
+ * Represents a temporary client configuration used exclusively for loading client details from
+ * inline YAML configuration. This class is intended as a temporary workaround to facilitate the
+ * deserialization of client configurations. Note: If you add any new fields to the CustodialClient
+ * or NonCustodialClient classes, ensure that you also add those fields here to maintain
+ * consistency.
+ */
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class TempClient {
```

Comment:
naming suggestion: 
```suggestion
public class RawClientConfig {
```

Reply:
Any feedback?

Reply:
I think the naming is fine. This object is for temp use. It will be removed once we done migration from Spring